### DISCUSSION TITLE
Are there security audits from 3rd parties?

### DISCUSSION THREAD

**Discussion by hpvd** [nats-io/nats-server#6231] [Q&A]
We are just evaluating Nats, Jetstream and Nex
-> Are there any known/public security audits provided by 3rd parties beside this one from 2018?
https://github.com/nats-io/nats-general/blob/main/reports/Cure53_NATS_Audit.pdf

PS:
Of course we found and highly appreciate

- the continuous internal work of @philpennock 
https://github.com/nats-io/nats-server/security

- and the basic information in this self assessment
https://tag-security.cncf.io/community/assessments/projects/nats/self-assessment/

**Comment by derekcollison** [comment]
Yes we have had several, will let @bruth get you the details.

**Reply by hpvd** [reply]
sounds good. Many thanks. Looking forward to it!

**Reply by bruth** [reply]
Hi @hpvd, we had a new security audit in 2024. It is being finalized for public consumption by the auditors, but if you email me (byron@nats.io) I can share the draft with a couple notes.
